dsgrid/config/time_dimension_base_config.py

```diff
@@ -254,26 +243,34 @@ def _align_time_interval_type(self, df, project_time_dim):
         time_col = time_col[0]
 
         if dtime_interval == ptime_interval:
-            return df
+            return df, False
 
         df = self._shift_time_interval(
             df, time_col, dtime_interval, ptime_interval, self.get_frequency()
         )
+        return self._handle_time_difference(df, project_time_dim, wrap_time), True
+
+    def _handle_time_difference(
+        self, df: DataFrame, project_time_dim: "TimeDimensionBaseConfig", wrap_time: bool
+    ) -> DataFrame:
         diff = self._time_difference(df, project_time_dim, difference="left")
         if diff:
+            if not wrap_time:
+                logger.warning("Wrapping time even though the config does not permit it.")
```

Comment:
Can we provide any more information in this warning message so the user can get a sense for what's happening?